src/App.java

```diff
@@ -1,34 +1,264 @@
+            System.out.println("I kept going until I accidentally knocked over a " + noun2 + ".");
+            System.out.println("I'll never forget that " + adj3 + " day. The day I became (a/an) " + noun3 + ".");
+            System.out.println(r+"==============");
+        } else if (choice == 3) { // story 3
+            // collect inputs (I thought about making a function like takeInput(adjective/verb) but I couldn't figure out variable creation
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj1 = getValidInput();
+
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj2 = getValidInput();
+            
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj3 = getValidInput();
+            
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun1 = getValidInput();
+            
+            System.out.print(g+"Sound:\n"+w+" ↳ ");
+            String sound1 = getValidInput();
+            
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun2 = getValidInput();
+            
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun3 = getValidInput();
+            
+            System.out.print(g+"Verb:\n"+w+" ↳ ");
+            String verb1 = getValidInput();
+            
+            System.out.print(g+"Past tense verb:\n"+w+" ↳ ");
+            String ptverb1 = getValidInput();
+            
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj4 = getValidInput();
+            
+            // print out madlibs
+            System.out.println("One " + adj1 + " night, my friends and I decided to explore the " + adj2 + " house at the end of the street.");
+            System.out.println("We heard that it was haunted by a " + adj3 + " " + noun1 + ".");
+            System.out.println("As we went inside, we heard a loud " + sound1 + ".");
+            System.out.println("Then, a " + noun2 + " came flying towards us.");
+            System.out.println("We ran upstairs and hid behind a " + noun3 + ", but the " + noun2 + " followed us.");
+            System.out.println("We decided to " + verb1 + " to try to stop it, but it just " + ptverb1 + ".");
+            System.out.println("It was so " + adj4 + ".");
+            System.out.println(r+"==============");
+        }
+    }
+
+    // I didn't want to keep writing "madlibs.nextLine.trim().toLowerCase();" so I made a function to take in user input, format, and make sure it's not empty
+    public String getValidInput() {
+        String input;
+        do {
+            input = madlibs.nextLine().trim().toLowerCase();
+            if (input.isEmpty()) {
+                System.out.print(g+"Input cannot be empty, please try again:\n"+w+" ↳ ");
+            }
+        } while (input.isEmpty());
         
+        return input;
     }
-    
   
+    public void printGoodbye() {
```

Comment:
This makes me think `printWelcome()` should have been its own method, but not a big deal